contrib/kind-helm.sh

```
@@ -332,7 +332,7 @@ parse_args() {
             -lr | --local-kind-registry )         KIND_LOCAL_REGISTRY=true
                                                   ;;
             -ep | --experimental-provider )       shift
-                                                  export KIND_EXPERIMENTAL_PROVIDER="$1"
+                                                  KIND_EXPERIMENTAL_PROVIDER=$1
```

Shellcheck (0.11.0)

[warning] KIND_EXPERIMENTAL_PROVIDER appears unused. Verify use (or export if used externally).

(SC2034)